NYTPhotoViewer/NYTPhotoViewerArrayDataSource.m

```diff
@@ -1,20 +1,14 @@
 //
-//  NYTPhotosDataSource.m
+//  NYTPhotoViewerArrayDataSource.m
 //  NYTPhotoViewer
 //
 //  Created by Brian Capps on 2/11/15.
 //
```

Comment:
Should there be a copyright statement here. Not sure what the protocol is for open source.